tests/integration/sentinel/schedules/outbound.spec.js

```diff
@@ -156,6 +174,13 @@ describe('Outbound', () => {
           'completed_tasks[1].type': 'outbound',
           'completed_tasks[1].name': 'also_working',
         });
+        // chai.expect(infoDocs[1]).to.nested.include({
+        //   _id: 'test-cron-info',
+        //   type: 'info',
+        //   doc_id: 'test-cron',
+        //   'completed_tasks[0].type': 'outbound',
+        //   'completed_tasks[0].name': 'working_with_cron',
+        // });
```

Comment:
Can you please delete the commented lines? And to unblock you I can create another issue to complete this scenario.